lib/multi.c

```diff
@@ -3047,10 +3051,10 @@ void Curl_multi_dump(struct Curl_multi *multi)
 }
 #endif
 
-uint32_t Curl_multi_max_http2_concurrent_streams
+size_t Curl_multi_max_concurrent_streams
 (struct Curl_multi *multi)
 {
   return multi ?
-    (multi->max_http2_concurrent_streams?
-    multi->max_http2_concurrent_streams:100) : 0;
+    (multi->max_concurrent_streams?
+    multi->max_concurrent_streams:100) : 0;
```

Comment:
`multi->max_concurrent_streams` is a `long` and this returns a `size_t` ? On some platforms like Windows, they are not of equal size.

Reply:
Its the same for max_host_connections, max_total_connections. May be I can type cast it to size_t, which would adjust according to architecture used.